<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>260</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>This Color</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/resource/this color.png</normaloff>:/resource/this color.png</iconset>
  </property>
  <widget class="QLabel" name="label_screen">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_color">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Pointed Color</string>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>150</width>
     <height>140</height>
    </rect>
   </property>
   <property name="title">
    <string>RGB</string>
   </property>
   <widget class="QProgressBar" name="progress_R">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;Red&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar::chunk {
background-color:rgba(255,0,0,0.5);
margin :2px;
border-radius:2px;
}

QProgressBar {
    text-align: center; 
	
}</string>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>200</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="format">
     <string>%v</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progress_G">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>45</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#00ff00;&quot;&gt;Green&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar::chunk {
background-color:rgba(0,255,0,0.5);
margin :2px;
border-radius:2px;
}

QProgressBar {
    text-align: center; 
	
}
QToolTip{
background-color: qlineargradient(spread:reflect, x1:1, y1:0.5, x2:1, y2:1, stop:0 rgba(50, 50, 50, 255), stop:1 rgba(80, 80, 80, 255));
border:1px solid #888;
}</string>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>200</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="format">
     <string>%v</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progress_B">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#0000ff;&quot;&gt;Blue&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar::chunk {
background-color:rgba(0,0,255,0.5);
margin :2px;
border-radius:2px;
}

QProgressBar {
    text-align: center; 
	
}</string>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>200</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="format">
     <string>%v</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_R">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>40</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>R: 255</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_G">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>100</y>
      <width>40</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>G: 255</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_B">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>40</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>B: 255</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_HEX_RGB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>#FFFFFF</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_color_R">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;Red&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_color_G">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#00ff00;&quot;&gt;Green&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolTip{
background-color: qlineargradient(spread:reflect, x1:1, y1:0.5, x2:1, y2:1, stop:0 rgba(50, 50, 50, 255), stop:1 rgba(80, 80, 80, 255));
border:1px solid #888;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_color_B">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#0000ff;&quot;&gt;Blue&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="zoom_in">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Zoom In</string>
   </property>
   <property name="text">
    <string>▶</string>
   </property>
  </widget>
  <widget class="QPushButton" name="zoom_out">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Zoom Out</string>
   </property>
   <property name="text">
    <string>◀</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_zoom_rate">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Zoom x2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>150</width>
     <height>140</height>
    </rect>
   </property>
   <property name="title">
    <string>CMYK</string>
   </property>
   <widget class="QProgressBar" name="progress_C">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#00ffff;&quot;&gt;Cyan&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar::chunk {
background-color:rgba(0,255,255,0.5);
margin :2px;
border-radius:2px;
}

QProgressBar {
    text-align: center; 
	
}
QToolTip{
background-color: qlineargradient(spread:reflect, x1:1, y1:0.5, x2:1, y2:1, stop:0 rgba(50, 50, 50, 255), stop:1 rgba(80, 80, 80, 255));
border:1px solid #888;
}</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="format">
     <string>%v%</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progress_Y">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot;color:#ffff00;&quot;&gt;Yellow&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar::chunk {
background-color:rgba(255,255,0,0.5);
margin :2px;
border-radius:2px;
}

QProgressBar {
    text-align: center; 
	
}
QToolTip{
background-color: qlineargradient(spread:reflect, x1:1, y1:0.5, x2:1, y2:1, stop:0 rgba(50, 50, 50, 255), stop:1 rgba(80, 80, 80, 255));
border:1px solid #888;
}</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="format">
     <string>%v%</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progress_M">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>45</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff00ff;&quot;&gt;Magenta&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar::chunk {
background-color:rgba(255,0,255,0.5);
margin :2px;
border-radius:2px;
}

QProgressBar {
    text-align: center; 
	
}</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="format">
     <string>%v%</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progress_K">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>95</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Black</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar::chunk {
background-color:rgba(0,0,0,0.5);
margin :2px;
border-radius:2px;
}

QProgressBar {
    text-align: center; 
	
}</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="format">
     <string>%v%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_color_C">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#00ffff;&quot;&gt;Cyan&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolTip{
background-color: qlineargradient(spread:reflect, x1:1, y1:0.5, x2:1, y2:1, stop:0 rgba(50, 50, 50, 255), stop:1 rgba(80, 80, 80, 255));
border:1px solid #888;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_color_Y">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffff00;&quot;&gt;Yellow&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolTip{
background-color: qlineargradient(spread:reflect, x1:1, y1:0.5, x2:1, y2:1, stop:0 rgba(50, 50, 50, 255), stop:1 rgba(80, 80, 80, 255));
border:1px solid #888;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_color_M">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff00ff;&quot;&gt;Magenta&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_color_K">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>95</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Black</string>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_C">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>120</y>
      <width>40</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>C:100</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Y">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>40</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Y:100</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_M">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>40</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>M:100</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_K">
    <property name="geometry">
     <rect>
      <x>118</x>
      <y>120</y>
      <width>40</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>K:100</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_cross">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Display &amp; Cross On/Off</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton::pressed{
background-color:rgba(0,0,0,0);
}
QPubhsutton{
background-color:rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>91</width>
     <height>91</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
